boefjes/boefjes/local.py

```diff
@@ -40,10 +42,14 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 class LocalBoefjeJobRunner(BoefjeJobRunner):
     def __init__(self, local_repository: LocalPluginRepository):
         self.local_repository = local_repository
+        self.scheduler_client = SchedulerAPIClient(str(settings.scheduler_api))
 
     def run(self, boefje_meta: BoefjeMeta, environment: Dict[str, str]) -> List[Tuple[set, Union[bytes, str]]]:
         logger.info("Running local boefje plugin")
 
+        task_id = str(boefje_meta.id)
+        self.scheduler_client.patch_task(task_id, TaskStatus.RUNNING)
+
```

Comment:
I don't know if adding this is advisable or desirable. I took note from the docker boefje runner. @Donnype

Reply:
The main issue was that the boefje and normalizer runner are not updating the task when they are processing the job. This change will make sure that it happens and that the duration calculation is done for a task

Reply:
@jpbruinsslot For the Docker setup there were indeed some limitation that meant that status handling had to happen in the runner. The initial setup that uses the local runner handles the status in the `app.py`, which I'd say for now is a better place to add this I think for the local setup.

Reply:
@Donnype updated, and added it to `app.py`. Would you please confirm that it is in the correct place?

Reply:
Looks good @jpbruinsslot !